api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -136,6 +136,14 @@ default AppendFiles newFastAppend() {
    */
   RewriteFiles newRewrite();
 
+  /**
+   * Create a new {@link RewriteManifests rewrite manifests API} to replace manifests for this
+   * table and commit.
+   *
+   * @return a new {@link RewriteManifests}
+   */
+  RewriteManifests newRewriteManifests();
```

Comment:
Can we change this to just `rewriteManifests` and use the verb?

I know this matches `newRewrite`, but that rewrite is applying rewrite changes, not actually doing the rewrite.

Reply:
Sure, this is done.